lib/remote/httpserverconnection.cpp

```diff
@@ -171,6 +171,40 @@ void HttpServerConnection::ProcessMessageAsync(HttpRequest& request)
 
 	HttpResponse response(m_Stream, request);
 
+	Array::Ptr headerAllowOrigin = ApiListener::GetInstance()->GetAccessControlAllowOrigin();
+
+	if (headerAllowOrigin->GetLength() != 0)
+	{
```

Comment:
Parenthesis for conditions, loops, etc. should be on the same line. This applies to all the other code changes made here :)